### DISCUSSION TITLE
Question: How do I add a custom alias for a typable command?

### DISCUSSION THREAD

**Discussion by dheerajbhaskar** [helix-editor/helix#6790] [Q&A]
# Summary
I wanted to add :ra as an alias for :reload-all. This is similar to :pwd for :show-directory. Is there a way to add aliases with a configuration file (not source code change)?

## Related question:
I use termux. The issue I'm trying to solve here is that helix's gutter doesn't update when I do a git commit. I do the git commit in another termux session. Gutter remains stale until I reload my opened buffers.

Is there a programmatic way to solve this? Maybe by interacting with running helix instance?

# Related source code for typed command aliases

https://github.com/helix-editor/helix/blob/7706ff77eb8aea9471af822355cad337707b9fc5/helix-term/src/commands/typed.rs

**Comment by the-mikedavis** [comment]
It isn't currently possible to rename typable commands. That is covered by https://github.com/helix-editor/helix/issues/4423

The gutter refresh will be automatic once file watching is implemented. Interacting with a running Helix instance from outside the process isn't possible currently but is covered by #312
